__tests__/articles.test.js

```diff
@@ -135,3 +135,87 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe.only("POST /api/articles/:article_id/comments", () => {
+  it("201: should post comment to comments data table with correct properties", () => {
+    const test_id = 1;
+    return request(app)
+      .post(`/api/articles/${test_id}/comments`)
+      .send({
+        username: "lurker",
+        body: "This is my comment",
+      })
+      .expect(201)
+      .then(({ body }) => {
+        const { comment } = body;
+        expect(comment).toMatchObject({
+          comment_id: expect.any(Number),
+          body: "This is my comment",
+          article_id: test_id,
+          author: "lurker",
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+        });
+      });
+  });
+  it("400: should respond with error message when passed invalid article_id", () => {
```

Comment:
I know you've changed this to be a 400 because you were having issues (see my main comment for more details on this), but the test description is incorrect. It should say its a valid but non-existent ID 👍